### DISCUSSION TITLE
[Suggestion] Add ability to copy text inside the youtube app

### DISCUSSION THREAD

**Discussion by AlphaOrBeta** [qnblackcat/uYouPlus#485] [Ideas]
It would be nice if we could copy text such as a comment or some text from video description such as a link that isn't a hyperlink or something different. I think that the ability to copy text would be pretty helpful in many occasions because I experienced the need to copy some text from youtube many times in the past.

**Comment by arichornlover** [comment]
This might be a good idea!
I’m tired of YouTube not letting us do it on the app, it just grabs the text as an image :(

**Comment by Skywalker8510** [comment]
that would actually be epic. iv had this problem way too often

**Comment by arichornlover** [comment]
Since iOS 16 developer beta 1 released, apple has added copying text in a video but I’ve wish google would add apple related features on the app :(

**Comment by AlphaOrBeta** [comment]
I didn't really mean to suggest adding support for copying text inside of videos, ok that's usefull. But what I meant for those who didn't get it by now is the ability to copy text inside of the application, places like the video description and comments.

**Comment by madison696969** [comment]
can't you just screenshot and use the text detection in the photos app?

**Comment by arichornlover** [comment]
> can't you just screenshot and use the text detection in the photos app?

Well yeah but it takes more time, screenshotting it, copying the text and deleting the screenshot isn’t justifiable until YouTube gives users an option to enable copy texts in videos.
